### ISSUE TITLE
Having the turn waiting on a leaver should force the leaver to concede

### ISSUE THREAD

**Issue by mx** [magefree/mage#5116]
Many EDH games have been ended because a player leaves without conceding, which leaves their state on the board. When something affects their state, the game then waits on them to move. But they aren't there. If the game reaches a state like this, the player should be automatically conceded.

**Comment by mx** [comment]
More accurately -- the player *does* get conceded, but their state remains.

Just happened to me. Player left w/ creatures on the board. I cast Terminus. They time out, get conceded. Despite this, xmage still waits for them to act. Thus the game cannot move forward.

**Comment by Zerrisx** [comment]
Duplicate of #4472 and #2423
